omise/setting.php

```diff
@@ -0,0 +1,86 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+class Setting extends PaymentModule
+{
+    protected $submit_action = 'omise_save_setting';
```

Comment:
@nimid ✋ more question.
>  it can not be changed and extended.

Why it need to be able to be changed? (not sure which case that we will change it.)
Also, which case that it need to be able to be extended?